src/app/app-routing.module.ts

```diff
@@ -50,6 +50,7 @@ const routes: Routes = [
     {path: 'storageDetails/:storageId', loadChildren: './business/delfin/storage-details/storage-details.module#StorageDetailsModule'},
     {path: 'modifyStorage/:storageId', loadChildren: './business/delfin/modify-storage/modify-storage.module#ModifyStorageModule'},
     {path: 'servicePlanManagement', loadChildren: './business/service-plan/service-plan.module#ServicePlanModule'},
+    {path: '**', redirectTo: '/home'}
```

Comment:
Redirection works but the css in the page is not changed.
When we are in any other page than the home page we have  white page background. 
On the home page we have a blue image background. 
When we redirect to home page in the code using the routerLink to navigate method we explicitly change this value and change the background. 
This check will need to be made in this case as well.

Reply:
this is a known issue on the soda dashboard. I checked without this branch this problem is still arising. However, this PR does not fix that this PR only fixes #653. for fixing it we can open a new PR.